app/src/main/java/com/spartak/recipesapp/ui/favorite_screen/FavoriteViewModel.kt

```diff
@@ -39,4 +49,8 @@ class FavoriteViewModel @Inject constructor(
     fun deleteRecipeInDb(recipe: Recipe) {
         recipeRepository.deleteFavoriteRecipe(recipe).applySchedulers(onSuccess = {}, onError = {})
     }
+
+    fun setSearchText(text: String) {
+        _searchText.postValue(text)
```

Comment:
simplify:

if (it.isBlank()) viewModel.fetchFavoriteRecipes()
                else viewModel.fetchSearchRecipes(it)